### PR TITLE
Muniyammal

### PR DESCRIPTION
**Pull request by Muniyammal**
function.js implemented.

### CODE REVIEW

File: muniyammal/funover.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*This program implements the function overloading
+*prints the standard rresult as output
+*/
+//overload the function 
+function bakya(a,b,c)
```
Comment (MurugalakshmiM) [inline muniyammal/funover.js:5]: naming convention is important. make use of proper name.

File: muniyammal/largest.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+var arr = "";
+function largest(arr) {
+	var large = arr[0];
+	for(var i=0; i<arr.length; i++){
```
Comment (MurugalakshmiM) [inline muniyammal/largest.js:4]: check var and let scope

File: muniyammal/largest.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+		if(large < arr[i]) {
+			large = arr[i];
+		}
+	}
+	return large;
+}
+
+var input ={name:"raju", notes: [10,20,30],topnotes:''};
+ input.topnotes = largest(input.notes);
```
Comment (MurugalakshmiM) [inline muniyammal/largest.js:13]: check mutable concept

File: muniyammal/bit.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/*This program performs a Bitwise operation
+*That displays a result to the standard output
+*/
+//comma operator when you want to include multiple expressions in a location that requires a single expression.
+let a,b,c,d,e,f,a1;
```
Comment (MurugalakshmiM) [inline muniyammal/bit.js:5]: make use of proper name for variable

File: muniyammal/func.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+/*This program performs a addition operation
+*prints the standard result as a output
+*/
+var height = 5;
```
Comment (MurugalakshmiM) [inline muniyammal/func.js:4]: practice to use variable types according to the version of ECMA script you are using.

File: muniyammal/mirror.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<html>
+  <head>
+     <title> mirror letters </title>
+  </head>
+ <body>
+    <script>
+            var str = "bakya";
```
Comment (MurugalakshmiM) [inline muniyammal/mirror.html:7]: what about capital one?.

Comment (Muniyammal) [reply muniyammal/mirror.html:7]: captial mirror letter program was pulled successfully

File: muniyammal/func.js (hunk 5 above)
Comment (Muniyammal) [reply muniyammal/func.js:4]: ok ,I'll  adapt

File: muniyammal/bit.js (hunk 4 above)
Comment (Muniyammal) [reply muniyammal/bit.js:5]: I'll change that variable name.

File: muniyammal/largest.js (hunk 3 above)
Comment (Muniyammal) [reply muniyammal/largest.js:13]: I dont get a clear idea of this concept.so you can please explain the concept ,

File: muniyammal/berth.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+const seat= require("readline")
+const s =seat.createInterface({
+    input : process.stdin,
+    output : process.stdout
+})
+s.question("enter the seat numner",(number)=>{
```
Comment (pbravichandhar) [inline muniyammal/berth.js:6]: Reduce logic

**Review by pbravichandhar — APPROVED** [review]

File: muniyammal/funover.js (hunk 1 above)
Comment (Muniyammal) [reply muniyammal/funover.js:5]: k akka correct panre

File: muniyammal/berth.js (hunk 7 above)
Comment (Muniyammal) [reply muniyammal/berth.js:6]: kk anna

**Pull request merged**